app/human-detector-app/classes/Camera.tsx

```diff
@@ -0,0 +1,55 @@
+import Group from './Group';
+
+export default class Camera {
+  //field
+  cameraId:string;
+  cameraName:string;
+  userId:string;
+
+  //constructor
+  constructor(userId:string, cameraName:string, cameraId:string) {
+      this.cameraId = cameraId;
+      this.cameraName = cameraName;
+      this.userId = userId;
+  }
+
+  set setCameraName(cameraName:string){
+      this.cameraName = cameraName;
+  }
+
+  renameCamera(name: string): Camera {
+    return new Camera('0', 'name', 'ID');
+  }
+
+  isCameraOnline(): boolean {
+    return true;
+  }
+
+}
+
+export function searchForCameras(): Camera[]{
```

Comment:
Is this meant to search for every camera? Searching for cameras by customer ID makes more sense to me, but I need to review the endpoints I exposed in the design doc

Reply:
It actually would be searching by customer ID I think that's what the endpoint was described as ill change that real quick

Reply:
Changed in f860a2ed0ef26c85b5a3ad180ea0752723e1921a